crates/sui-graphql-rpc/src/types/transaction_exec.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::*;
+
+#[derive(SimpleObject, Clone)]
+pub(crate) struct ExecutionResult {
+    /// The errors field captures any errors that occurred during execution
```

Comment:
If the follow up for getting `TransactionBlockEffects` included in here is not immediate, could you add a TODO here and record it in the launch tracker?